altos-rust/Makefile

```diff
@@ -90,11 +90,11 @@ test:
 
 doc:
 	@mkdir -p $(build_path)
-	@$(cargo) doc $(doc_args)
-	@cp -r $(doc_target_path) $(doc_build_path)
+	@cd port/cortex-m0; \
```

Comment:
I don't see any docs being generated for the usart directory when i run `make doc`. I think it has something to do with the conditional compilation of your serial code.